bigchaindb/core.py

```diff
@@ -136,11 +138,54 @@ def write_transaction(self, signed_transaction, durability='soft'):
         signed_transaction = deepcopy(signed_transaction)
         # update the transaction
         signed_transaction.update({'assignee': assignee})
+        signed_transaction.update({'assignment_timestamp': time()})
 
         # write to the backlog
         response = r.table('backlog').insert(signed_transaction, durability=durability).run(self.conn)
         return response
 
+    def reassign_transaction(self, transaction, durability='hard'):
```

Comment:
Makes sense.

There is also the problem that since the assignment_timestamp is not updated then the stale monitor of that node will be constantly retrieving the same transactions. But I guess that if this is a problem then the node should increase the reassign_timeout